tests/testthat/test_covariance_plot.R

```diff
@@ -0,0 +1,32 @@
+test_that("h_get_timepoint_vars works as expected with string specified", {
+  data("mmrm_test_data")
+  data_wide <- dcast(mmrm_test_data, USUBJID + ARMCD ~ AVISIT, value.var = "FEV1")
```

Comment:
please use 
```suggestion
  data_wide <- maditr::dcast(mmrm_test_data, USUBJID + ARMCD ~ AVISIT, value.var = "FEV1")
```
here and elsewhere to use the package function